tools/multibuild/Dockerfile.package

```diff
@@ -0,0 +1,57 @@
+# Dockerfile.package
+# A dockerfile for packaging SSH No Ports releases using docker buildx
+
+FROM atsigncompany/buildimage:3.5.2_3.6.0-149.3.beta@sha256:df67b9e3271381fc0c5b20e7350cf4de8dad6ac62e075b49b1a866c49af47409 AS build
+# Using atsigncompany/buildimage until official dart image has RISC-V support
+WORKDIR /noports
+
+# install node for later (keep at the top file to increase cache hits)
+# hadolint ignore=DL3008
+RUN apt-get update; \
+  apt-get install -y --no-install-recommends npm
+
+COPY . .
+
+# Build packages/dart/sshnoports
+WORKDIR /noports/packages/dart/sshnoports
```

Comment:
Changed some pathing here, the new build context is the root of the repo instead of packages/dart so that the apps directory is also within the context (docker context doesn't work with `../` paths)